README.md

```diff
@@ -243,6 +244,27 @@ You may want to check other ways of authentication
 Doorkeeper support multiple languages. See language files in
 [the I18n repository](https://github.com/doorkeeper-gem/doorkeeper-i18n).
 
+### Rake Tasks
+
+If you are using `rake`, you can load rake tasks provided by this gem, by adding
+the following line to your `Rakefile`:
+
+```ruby
+Doorkeeper::Rake.load_tasks
+```
+
+#### Cleaning up
+
+If you are concerned with some of the tables added by Doorkeeper getting too large
+over time, you can regularly run the following rake task to remove stale entries
+from the database:
+
+```rake
+rake doorkeeper:db:cleanup
+```
+
+Note that by default those records are retained to allow stricter auditing.
```

Comment:
Actually I assume that this is the reason. Feel free to fill me in on a better explanation :)